docs/resources/ccm_certificate_push.md

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Cloud Certificate Manager (CCM)"
+---
+
+# huaweicloud_ccm_certificate_push
+
+Push ceritificate to elb or waf within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "certificate_id" {}
+
+variable "project_name" {}
+
+resource "huaweicloud_ccm_certificate_push" "test"{
+  region         = "cn-north-4"
+  certificate_id = var.certificate_id
+  service        = "ELB"
+  targets {
+    project_name = var.project_name
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the certificate region. Changing this creates a new
+  private certificate resource. Now only support cn-north-4 (china) and ap-southeast-1 (international)
+
+* `certificate_id` - (Required, String, ForceNew) Specifies the certificate which will be pushed.
+  Changing this parameter will create a new resource.
+
+* `service` - (Required, String, ForceNew) Specifies the service which certificate be pushed to. Changing this creates
+  a new resource.
+
+* `targets` - (Required, List) Specifies the project which certificate be pushed to.
+  The [targets](#block-targets) structure is documented below.
+
+<a name="block-targets"></a>
+The `targets` block supports:
+
+* `project_name` - (Required, String) Specifies the project which certificate be pushed to.
```

Comment:
please add the limitation: Only the **default** projects with the same region name are supported.